router/batchrouter/asyncdestinationmanager/yandex-metrica-offline-events/yandexmetricaofflineevents.go

```diff
@@ -0,0 +1,161 @@
+package yandexmetricaofflineevents
```

Comment:
prefer a small name for the package and the folder name:

```suggestion
package yandexmetrica
```

Also the folder name should be `yandexmetrica` for consistency